documentation/Configuration.md

```diff
@@ -6,6 +6,7 @@ StyleCop Analyzers is configured using two separate mechanisms: code analysis ru
 
    * Enable and disable individual rules
    * Configure the severity of violations reported by individual rules
+   * migrate from old **StyleCop.Settings** to new .ruleset format (see [Migrate.md](documentation/Migrate.md))
```

Comment:
:thought_balloon: This bulleted list is intended to describe the functionality of the new rule set files, but this new item describes something else. A better placement for this as part of the descriptive text outside of the list itself.